content/blog/2020-06-08-june-20-heartbeat.md

```diff
@@ -0,0 +1,213 @@
+---
+title: June '20 Heartbeat
+date: 2020-06-08
+description: |
+  Catch up on new DVC releases, talks, and projects in our community. 
+  This month, learn about finishing touches on DVC 1.0, DVC in biomedical
+  research, recommended reading and upcoming MLOps talks.
+descriptionLong: |
+  Catch up on new DVC releases, talks, and projects in our community. 
+  This month, learn about finishing touches on DVC 1.0, DVC in biomedical
+  research, recommended reading and upcoming MLOps talks.
+picture:
```

Comment:
it fails to deploy it w/o the image, it's hard to review the PR - please add any image for now